main.go

```diff
@@ -165,11 +165,12 @@ func _main(ctx context.Context) error {
 			}
 		}
 		dbi := dcrpg.DBInfo{
-			Host:   pgHost,
-			Port:   pgPort,
-			User:   cfg.PGUser,
-			Pass:   cfg.PGPass,
-			DBName: cfg.PGDBName,
+			Host:         pgHost,
+			Port:         pgPort,
+			User:         cfg.PGUser,
+			Pass:         cfg.PGPass,
+			DBName:       cfg.PGDBName,
+			QueryTimeout: time.Duration(cfg.PGQueryTimeout*1e3) * time.Millisecond,
```

Comment:
If you converted `cfg.PGQueryTimeout` to be of type `time.Duration` then all you would need to do here is 
```golang
QueryTimeout: cfg.PGQueryTimeout,
```

Reply:
converting `cfg.PGQueryTimeout` to time.duration might not be that straight forward so you could do away with this.

Reply:
I'll look into this, but my concern was not so much marshal/unmarshal but how the go-flags will read from the command line and what is valid syntax.